compiler/runtime/OMRCodeCache.cpp

```diff
@@ -1275,8 +1276,8 @@ void
 OMR::CodeCache::dumpCodeCache()
    {
    printf("Code Cache @%p\n", this);
-   printf("  |-- warmCodeAlloc          = 0x%08x\n", _warmCodeAlloc );
-   printf("  |-- coldCodeAlloc          = 0x%08x\n", _coldCodeAlloc );
+   printf("  |-- warmCodeAlloc          = 0x%08" OMR_PRIxPTR "\n", (uintptr_t)_warmCodeAlloc );
+   printf("  |-- coldCodeAlloc          = 0x%08" OMR_PRIxPTR "\n", (uintptr_t)_coldCodeAlloc );
```

Comment:
These casts match the surrounding code, so I figure I'll leave them? Or should these use reinterpret too??

Reply:
I'd leave them. Other parts of the compiler use C-style casts too so I don't think it's worth changing these. I only insist on C++ style casts in tests, since that all "new" code. :slightly_smiling_face: